File: example/java/jimmer-sql-graphql/src/main/java/org/babyfish/jimmer/sql/example/graphql/entities/CommonEntity.java
package org.babyfish.jimmer.sql.example.graphql.entities;

import org.babyfish.jimmer.sql.MappedSuperclass;

import java.time.LocalDateTime;

/*
 * set CommonEntityDraftInterceptor
 */
@MappedSuperclass
public interface CommonEntity {

    LocalDateTime createdTime();

    LocalDateTime modifiedTime();
}

File: example/kotlin/jimmer-sql-graphql-kt/src/main/kotlin/org/babyfish/jimmer/example/kt/graphql/entities/CommonEntity.kt
package org.babyfish.jimmer.example.kt.graphql.entities

import org.babyfish.jimmer.sql.MappedSuperclass
import java.time.LocalDateTime

/*
 * see CommonEntityDraftInterceptor
 */
@MappedSuperclass
interface CommonEntity {

    val createdTime: LocalDateTime

    val modifiedTime: LocalDateTime
}